vue-app/src/components/RecipientSubmissionWidget.vue

```diff
@@ -204,13 +203,6 @@ export default class RecipientSubmissionWidget extends Vue {
       currentUser
     ) {
       try {
-        if (DateTime.now() >= currentRound.votingDeadline) {
```

Comment:
I believe we want this check if there is a current round. So logic should be something like:
```
if (!!currentRound && DateTime.now() >= currentRound.votingDeadline) {
```